src/ui/iconloader.cpp

```diff
@@ -31,10 +33,20 @@ void IconLoader::Init() {
 QIcon IconLoader::Load(const QString& name) {
   QIcon ret;
 
+  // If the icon name is empty 
   if (name.isEmpty()) return ret;
 
+  // Try to load icons from the custom icon location initially
+  const QString locate(QDir::toNativeSeparators(Utilities::GetConfigPath(Utilities::Path_Icons) + "/icons/%1x%2/%3.png"));
```

Comment:
I don't think you need `toNativeSeparators`, just stick to `/` everywhere and it will just work with Qt functions.